test/test.js

```diff
@@ -250,6 +258,27 @@ describe('cypress-terminal-report', () => {
     });
   }).timeout(60000);
 
+  it('Should not generate/print to output files.', async () => {
+    const outRoot = path.join(__dirname, 'output');
```

Comment:
Duplicated code: extract the output cleanup in an arrow function outside of the tests.